test/integration/targets/ios_acl/tests/cli/replaced.yaml

```diff
@@ -0,0 +1,72 @@
+---
+- debug:
+      msg: "START Replaced ios_acl state for integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+    - name: Replaces device configuration of listed interfaces with provided configuration
+      ios_acl: &replaced
+        config:
+          - afi: ipv4
+            acls:
+              - name: 110
+                aces:
+                  - grant: deny
+                    protocol_options:
+                      tcp:
+                        syn: true
+                    source:
+                      address: 192.0.2.0
+                      wildcard_bits: 0.0.0.255
+                    destination:
+                      address: 192.0.3.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: www
+                    dscp: ef
+                    ttl:
+                      eq: 10
+              - name: 150
+                aces:
+                  - grant: deny
+                    sequence: 20
+                    protocol_options:
+                      tcp:
+                        syn: true
+                    source:
+                      address: 198.51.100.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    destination:
+                      address: 198.51.110.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    dscp: ef
+                    ttl:
+                      eq: 10
+        state: replaced
+      register: result
+
+    - assert:
+        that:
+          - "result.commands|length == 5"
```

Comment:
Please assert the before and after list of dicts.

Reply:
same as above